storage.py

```
@@ -0,0 +1,98 @@
+            local_path: Local path to save the file
+            
+        Returns:
+            bool: True if download was successful
+        """
+        try:
+            # Create directory if it doesn't exist
+            os.makedirs(os.path.dirname(local_path), exist_ok=True)
+            
+            # Direct download from public URL
+            url = f"{self.public_url}/{object_name}"
+            response = requests.get(url)
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)